block/public.go

```diff
@@ -35,20 +35,19 @@ func NopMetrics() *Metrics {
 // DAClient is the interface representing the DA client for public use.
 type DAClient = da.Client
 
-// NewDAClient creates a new DA client with configuration
+// NewDAClient creates a new DA client backed by the blob JSON-RPC API.
 func NewDAClient(
-	daLayer coreda.DA,
+	blobRPC *blobrpc.Client,
 	config config.Config,
 	logger zerolog.Logger,
 ) DAClient {
 	return da.NewClient(da.Config{
```

Comment:
The max blob size will always be empty, which will fall back to the default value.
This may be fine, but don't we want to put it back as an arg?

Reply:
bump